test/suites/integration/ami_test.go

```diff
@@ -195,9 +184,10 @@ var _ = Describe("AMI", func() {
 		})
 		It("should support Custom AMIFamily with AMI Selectors", func() {
 			nodeClass.Spec.AMIFamily = &v1beta1.AMIFamilyCustom
+			al2AMI := env.GetCustomAMI("/aws/service/eks/optimized-ami/%s/amazon-linux-2/recommended/image_id", 1)
 			nodeClass.Spec.AMISelectorTerms = []v1beta1.AMISelectorTerm{
 				{
-					ID: customAMI,
+					ID: al2AMI,
```

Comment:
Why are we changing it from custom?

Reply:
This is still the custom AMI family. The `customAMI` is just the preselected AMI at the top of the file for all of the tests which is AL2023 now, not AL2. We can't generate a working AL2023 bootstrap script with the information known to the E2E test (need cluster CIDR and the CABundle).